client/src/components/RatingTile.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+
+const RatingTile = ({ id, content, activityId }) => {
+  const [ratings, setRatings] = useState({});
+  const [allRatings, setAllRatings] = useState([]);
+
+  const postRating = async (reviewId, selectedRating) => {
+    try {
+      const response = await fetch(`/api/v1/activities/${activityId}/reviews/rating`, {
```

Comment:
better that this is posted to just `/api/v1/ratings`, particularly with activities not being immediately relevant

this should then be in its own router